project1/styles.css

```diff
@@ -0,0 +1,60 @@
+h1 {
+    font-size: 70px;
+    text-align: center;
+    background-color: darkgrey;
+    font-family: 'IM Fell French Canon SC', serif;
+    color: #034247;
+
+}
+
+h2 {
+    font-size: 20px;
+    text-align: center;
+    background-color: darkgrey;
+}
+
+body {
+    margin: 0 auto;
+    max-width: 800px;
+    background-color: darkgrey;
+    font-family: 'Abel', sans-serif;
+    color: black;
+}
+
+hr {
+    width: 100%;
+    height: 2px;
+    background: #fff;
+}
+
+div {
+    text-align: center;
```

Comment:
This alignment rule gets overwritten by the styles on your "sub-border" class on line 41. You can instead delete _both_ `text-align` rules (the `div` is `text-align: left;` by default 👍 ) and add `text-align: center;` to your `.star` class instead (since that's the only bit here that should be centered).
```suggestion
```